components/brave_wallet/browser/swap_service.cc

```diff
@@ -325,8 +316,7 @@ void SwapService::OnGetPriceQuote(GetPriceQuoteCallback callback,
   if (!api_request_result.Is2XXResponseCode()) {
     if (auto swap_error_response =
             ParseSwapErrorResponse(api_request_result.value_body())) {
-      std::move(callback).Run(nullptr, std::move(swap_error_response),
-                              base::EmptyString());
```

Comment:
Using `""` as unsure if `base::EmptyString()` is the best option here, following the [docs](https://chromium.googlesource.com/chromium/src/+/master/base/strings/string_util.h#288).